src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/ScheduledReleaseFunction.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Functions;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+using Microsoft.Azure.Storage.Queue;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Functions
+{
+    public class ScheduledReleaseFunction
```

Comment:
Would be good to get a comment at the top here to explain the function of this class

Reply:
It's been renamed to PublishReleasesFunction now. It publishes all release scheduled to be published.